Tools/ci/build.sh

```diff
@@ -2,7 +2,7 @@
 set -ex # exit with nonzero exit code if anything fails
 
 # Build times benefit from parallel building
-export MAKE_PARALLEL="make -j3"
+export MAKE_PARALLEL="make -j$(nproc)"
```

Comment:
@slaff This surely won't work?

Reply:
My bad. Thanks.

Reply:
Think I'm going slightly potting switching between linux/windows all the time...